File: B/PHP/TimSepakBola.php
<?php

class TimSepakBola {
	// Atribut
	private $namaTim;		// Nama tim
	private $negara;		// Asal negara
	private $tahun;			// Tahun berdiri
	private $namaPemain;	// Nama pemain

	// Konstruktor
	public function __construct() {
		$this->namaTim = "";
		$this->negara = "";
		$this->tahun = 0;
		$this->namaPemain = [];
	}

	// Set & Get

	// Nama Tim
	public function setNamaTim($namaTim) {
		$this->namaTim = $namaTim;
	}
	public function getNamaTim() {
		return $this->namaTim;
	}

	// Asal Negara
	public function setNegara($negara) {
		$this->negara = $negara;
	}
	public function getNegara() {
		return $this->negara;
	}

	// Tahun Berdiri
	public function setTahun($tahun) {
		$this->tahun = $tahun;
	}
	public function getTahun() {
		return $this->tahun;
	}

	// Nama Pemain
	public function setNamaPemain($namaPemain) {
		$this->namaPemain[] = $namaPemain;
	}
	public function getNamaPemain($i) {
		return $this->namaPemain[$i];
	}

	// Destruktor
	function __destruct() {}
}

?>

File: B/Java/TimSepakBola.java
public class TimSepakBola {
	// Atribut
	private String namaTim;			// Nama tim
	private String negara;			// Asal negara
	private int tahun;				// Tahun berdiri
	private String namaPemain[];	// Nama pemain

	// Konstruktor
	public TimSepakBola(int jumlahPemain) {
		this.namaTim = "";
		this.negara = "";
		this.tahun = 0;
		this.namaPemain = new String[jumlahPemain];
	}

	/* Set & Get*/

	// Nama tim
	public void setNamaTim(String namaTim) {
		this.namaTim = namaTim;
	}
	public String getNamaTim() {
		return this.namaTim;
	}

	// Asal negara
	public void setNegara(String negara) {
		this.negara = negara;
	}
	public String getNegara() {
		return this.negara;
	}

	// Tahun berdiri
	public void setTahun(int tahun) {
		this.tahun = tahun;
	}
	public int getTahun() {
		return this.tahun;
	}

	// Nama pemain
	public void setNamaPemain(String namaPemain[], int jumlahPemain) {
		for(int i = 0; i < jumlahPemain; i++) {
			this.namaPemain[i] = namaPemain[i];
		}
	}
	public String getNamaPemain(int pemain) {
		return this.namaPemain[pemain];
	}
}